pages/apim/3.x/event-native-apim/event-native-apim-example-use-cases.adoc

```diff
@@ -0,0 +1,140 @@
+[[event-native-apim-example-use-cases]]
+= Event-native API Management example use cases
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/event_native_apim_example_use_cases.html
+:page-folder: apim/event-native-apim
+:page-layout: apim3x
+
+[label label-version]#New in version 3.20.0#
+[label label-version]#BETA release#
+
+[NOTE]
+====
+This document assumes you are familiar with synchronous APIs, asynchronous APIs, and the OpenAPI specification.
+====
+
+
+This page describes some example use cases for event-native API management.
+
+== Prerequisites
+
+=== Kafka
+
+These examples all depend on Kafka. To set up a Kafka Docker container with the link:https://github.com/tchiotludo/akhq[AKHQ] UI, follow these steps.
+
+1. Download `link:https://raw.githubusercontent.com/tchiotludo/akhq/master/docker-compose.yml[docker-compose.yml]`.
+
+2. Run it locally.
++
+[source, bash]
+----
+docker-compose pull
+docker-compose up
+----
+
+You can now access the AKHQ UI at http://localhost:8080.
+
+===  Enabling event-native API management
+
+To run these examples, you must set the `gravitee_api_jupiterMode_enabled` environment variable to `true` on the Management API and the API Gateway.
+
+=== Postman Collections
+
+These examples use the link:https://github.com/gravitee-io/postman-collections[Gravitee Postman Collections].
+
+These collections use four variables.
+
+[cols="1,1,1", options="header"]
+|===
+| Variable 
+| Description 
+| Example
+
+| `management_host`
+| The host for the Management API
+| `http://localhost:8083` 
+
+| `management_username`
+| The username for a management user
+| `admin`
+
+| `management_password`
+| The password for `management_username`
+| 
+
+| `gateway_host`
+| The gateway host
+| `http://loalhost:8082`
+|===
+
+== Data ingestion
```

Comment:
Some explanation could be great on the different use case achievable thank to api v4.

Reply:
Yes, I will add this in the next iteration.

Reply:
I have added a diagram showing some more detail here.